plugins/woocommerce/client/admin/webpack.config.js

```diff
@@ -188,30 +188,7 @@ const webpackConfig = {
 			new ForkTsCheckerWebpackPlugin(),
 		! process.env.STORYBOOK &&
 			new TypeScriptWarnOnlyWebpackPlugin( [
-				// these are the errors that have been converted into warnings during the react-18 upgrade
-				// the number is the original number of instances of that error when this comment is written
-				// hopefully this information helps prioritize fixing the errors :)
-				'TS2349', // ~107: This expression is not callable.
-				'TS2554', // ~60: Expected 2 arguments, but got 1.
-				'TS2339', // ~51: Property 'getActivePlugins' does not exist on type 'never'.
-				'TS7006', // ~38: Parameter implicitly has an 'any' type.
-				'TS2322', // ~31: Type mismatch in assignment.
-				'TS2503', // ~27: Cannot find namespace 'Slot'.
-				'TS2559', // ~11: Type mismatch in object property.
-				'TS2786', // ~10: Component cannot be used as a JSX element.
-				'TS2604', // ~10: JSX element type has no construct or call signatures.
-				'TS2345', // ~7: Argument type mismatch.
-				'TS2741', // ~5: Missing required property in type.
-				'TS2305', // ~4: Module has no exported member.
-				'TS2347', // ~3: Untyped function calls may not accept type arguments.
-				'TS2769', // ~2: No overload matches this call.
-				'TS2558', // ~2: Unexpected number of type arguments.
-				'TS2344', // ~2: Type does not satisfy a specific constraint.
-				'TS18046', // ~2: Variable is of type 'unknown'.
-				'TS2677', // 1: Type predicate's type must be assignable to its parameter's type.
-				'TS2571', // 1: Object is of type 'unknown'.
-				'TS2352', // 1: Invalid type conversion.
-				'TS18049', // 1: Nullable or undefined value encountered.
+				/* specify TS error codes here, e.g. 'TS2322' */
```

Comment:
We can remove the `TypeScriptWarnOnlyWebpackPlugin` entry here as well - it was introduced only to support react migration, and the build should be a bit faster without it as well.

Reply:
Thanks for the suggestion! @kalessil I've removed the TypeScript WarnOnlyWebpackPlugin entry and also removed the related TypeScript plugins from internal-style-build. f97ad358554a50e8c5b7a9d46f6679553562af07

Reply:
Perfect, thank you @chihsuan!